roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -10,11 +10,10 @@
   import_tasks: kubeadm-migrate-certs.yml
   when: old_apiserver_cert.stat.exists
 
-- name: kubeadm | Check apiserver key
+- name: kubeadm | Check serviceaccount key
   stat:
     path: "{{ kube_cert_dir }}/apiserver.key"
```

Comment:
task is changed, but you check for the serviceaccount key, yet you check `{{ kube_cert_dir }}/apiserver.key` is this correct?

Reply:
actually it should be serviceaccount key. This is because we need to know if the serviceaccount key changed to determine if we need to rotate every serviceaccount token. apiserver key can change as often as you like.